services/QuillLMS/config/application.rb

```diff
@@ -33,18 +29,6 @@ class Application < Rails::Application
       fog_directory: ENV.fetch('FOG_DIRECTORY', 'empirical-dev')
     }
 
-    # Custom directories with classes and modules you want to be autoloadable.
```

Comment:
This autoload path can be remove because
> [Zeitwerk] By default, the autoload paths of an application consist of all the subdirectories of app that exist when the application boots ---except for assets, javascript, and views--- plus the autoload paths of engines it might depend on.

https://guides.rubyonrails.org/v7.0/autoloading_and_reloading_constants.html#config-autoload-paths